bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -270,6 +209,8 @@ def _row_tuple_from_json(row, schema):
 
 def _rows_from_json(values, schema):
     """Convert JSON row data to rows with appropriate types."""
+    from google.cloud.bigquery import Row
```

Comment:
I am :-1: on this circular import, but hesitate to tell you "you must fix it". Though I wish you would.

Reply:
I created a new issue for this: https://github.com/GoogleCloudPlatform/google-cloud-python/issues/4305